spikeinterface/sortingcomponents/peak_detection.py

```diff
@@ -359,14 +347,263 @@ def _numba_detect_peak_neg(traces, traces_center, peak_mask, exclude_sweep_size,
                         continue
                     for i in range(exclude_sweep_size):
                         if chan_ind != neighbour:
-                            peak_mask[s, chan_ind] &= traces_center[s,
-                                                                    chan_ind] <= traces_center[s, neighbour]
-                        peak_mask[s, chan_ind] &= traces_center[s,
-                                                                chan_ind] < traces[s + i, neighbour]
-                        peak_mask[s, chan_ind] &= traces_center[s,
-                                                                chan_ind] <= traces[exclude_sweep_size + s + i + 1, neighbour]
+                            peak_mask[s, chan_ind] &= traces_center[s, chan_ind] <= traces_center[s, neighbour]
+                        peak_mask[s, chan_ind] &= traces_center[s, chan_ind] < traces[s + i, neighbour]
+                        peak_mask[s, chan_ind] &= traces_center[s, chan_ind] <= traces[exclude_sweep_size + s + i + 1, neighbour]
                         if not peak_mask[s, chan_ind]:
                             break
                     if not peak_mask[s, chan_ind]:
                         break
         return peak_mask
+
+
+class DetectPeakLocallyExclusiveOpenCL:
```

Comment:
Is the locally exclusive algorithm the same? Then in my opinion this should be called with `method='locally_exclusive', engine='opencl'`

Reply:
Yes the same.

Reply:
I known but I need to modify more stuff deeply. For now it is only a test